### PR TITLE
Fix: Change -p option to reference the directory path of compile_commands.json instead of the file path

### PR DESCRIPTION
**Pull request by JonathanFoo0523**
To fix issue [211](https://github.com/mc-imperial/dredd/issues/211)

### CODE REVIEW

**Comment by afd** [comment]
@JonathanFoo0523 can you rebase this over main and attend to the comment? Then I can take a final look and hopefully merge.

**Comment by afd** [comment]
@JonathanFoo0523 Can you take another look at this PR please?

**Comment by JonathanFoo0523** [comment]
@afd Done. Sorry for the late response. I overlooked this thread.

File: README.md [hunk 1]
```diff
@@ -135,10 +135,10 @@ All tests should pass.
 To mutate all of the `.cc` files in the library use the following command:
 
 ```
-${DREDD_EXECUTABLE} -p build/compile_commands.json math/src/*.cc --mutation-info-file mutant-info.json
+${DREDD_EXECUTABLE} -p build math/src/*.cc --mutation-info-file mutant-info.json
 ```
 
-The `-p` option allows the compilation database generated by CMake above to be passed to Dredd.
+The `-p` option allows the compilation database generated by CMake above to be passed to Dredd: the `compile_commands.json` file in build will be used as a compilation database.
```
Comment (afd) [inline README.md:141]: I'm being pedantic, but can you put backticks around `build` when you say "... in `build` will be used"?

**Review by afd — CHANGES_REQUESTED** [review]
One tiny change please.

File: README.md (hunk 1 above)
Comment (JonathanFoo0523) [reply README.md]: Fixed on [0aa58c8](https://github.com/mc-imperial/dredd/pull/212/commits/0aa58c8db555f37521b51dd6ec3d3a368840a906)

**Pull request merged**